test/unit/controllers/ileQuery/IleQueryControllerSpec.scala

```diff
@@ -140,7 +140,7 @@ class IleQueryControllerSpec extends ControllerLayerSpec with MockIleQueryCache
 
       "provided with correct DUCR" should {
 
-        val correctDucr = "9GB123456-QWERTY7890"
+        val correctDucr = "9GB123456789012-QWERTY7890"
```

Comment:
Would it be worth updating `CommonTestData` ? There are couple DUCRs there.

Reply:
Let me take a look, tests passed so I assume rest are correct

Reply:
Yeah, CommonTestData has correct DUCR